.gitattributes

```diff
@@ -1,2 +1,3 @@
 dist/trust-min.js filter=lfs diff=lfs merge=lfs -text
 dist/trust-min.js.map filter=lfs diff=lfs merge=lfs -text
```

Comment:
why do we have different file for iOS and Android?

Reply:
yeah, I also wondered, and it turns out jitpack doesn't support git lfs

Reply:
android package manager sucks, I will let @madcake  to solve this